src/platform/unittest/DataPathTest.cpp

```diff
@@ -696,12 +696,6 @@ TEST_F(DataPathTest, InitializeInvalid)
         ASSERT_EQ(QUIC_STATUS_INVALID_PARAMETER, Datapath.GetInitStatus());
         ASSERT_EQ(nullptr, Datapath.Datapath);
     }
-    if (!UseDuoNic) {
```

Comment:
This test case is invalid if you have XDP installed locally (which I do).

Reply:
WOW